app/assets/stylesheets/header.css

```diff
@@ -0,0 +1,144 @@
+body {
+  font-family: 'poppins', sans-serif;
+}
+
+header {
+  width: 100%;
+  background: none;
+  padding: 0 20px;
+}
+
+.menu {
+  background-color: #f5f5f5;
+}
```

Comment:
Great job so far @Gmanlove,
So there seems to  be a problem while clicking the foods link in the mobile menu!
![problem](https://github.com/codedit334/RoR_Group_Project/assets/114868555/0995e729-c14e-4792-9890-5e688361c99e)

You can fix that by adding `z-index: 1000;` to the .menu